prediction_market_agent/run_agent.py

```diff
@@ -67,6 +72,7 @@ class RunnableAgent(str, Enum):
     RunnableAgent.mech_prediction_request_reasoning: DeployablePredictionRequestReasoningAgent,
     RunnableAgent.mech_prediction_url_cot: DeployablePredictionUrlCotAgent,
     RunnableAgent.mech_prediction_with_research_bold: DeployablePredictionWithResearchBoldAgent,
+    RunnableAgent.social_media: DeployableSocialMediaAgent,
```

Comment:
Wasn't this merged already in https://github.com/gnosis/prediction-market-agent/pull/158/files? Idk why Github doesn't throw conflict 🤔